MODULE.bazel

```diff
@@ -0,0 +1,91 @@
+module(
+    name = "protoc-gen-validate",
+    compatibility_level = 1,
+    repo_name = "com_envoyproxy_protoc_gen_validate",
+)
+
+bazel_dep(
+    name = "bazel_skylib",
+    version = "1.8.1",
+)
+bazel_dep(
+    name = "gazelle",
+    version = "0.46.0",
+    repo_name = "bazel_gazelle",
+)
+bazel_dep(
+    name = "protobuf",
+    version = "33.0",
+    repo_name = "com_google_protobuf",
+)
+bazel_dep(
+    name = "re2",
+    version = "2024-07-02.bcr.1",
+    repo_name = "com_googlesource_code_re2",
+)
+bazel_dep(
+    name = "rules_cc",
+    version = "0.2.13",
+)
+bazel_dep(
+    name = "rules_go",
+    version = "0.58.3",
```

Comment:
You might want to update to 0.59.0, see https://github.com/bazelbuild/bazel-central-registry/pull/6780